### ISSUE TITLE
When trying to override, getting errors

### ISSUE THREAD

**Issue by meskarune** [nanotech/jellybeans.vim#73]
I added the following to my vimrc from your readme file:

```
128   let g:jellybeans_overrides = {                                                  
129       'background': { 'ctermbg': 'none', '256ctermbg': 'none' },                  
130   }
```

Then when I start vim I get these errors:

```
Error detected while processing /home/meskarune/.vimrc:
line  128:
E15: Invalid expression: 
E15: Invalid expression: {
line  129:
E20: Mark not set
line  130:
E492: Not an editor command: }
```

If I change line 128 to this:

```128 let g:jellybeans_overrides = {'background': { 'ctermbg': 'none', '256ctermbg': 'none' },} ```

vim runs without errors but the colors are not changed from the default. Am I doing something wrong?

**Comment by nanotech** [comment]
Vimscript doesn't implicitly continue unclosed expressions across lines, unlike most other languages. Subsequent lines need to be prefixed with a [line-continuation backslash](https://github.com/vim/vim/blob/9271d058c92c94b696eed5da24a69c077f42bc91/runtime/doc/repeat.txt#L416):

    let g:jellybeans_overrides = {
    \    'background': { 'ctermbg': 'none', '256ctermbg': 'none' },
    \}

As for the colors changing, what a background set to `none` will look like depends on the terminal you use and the colors it has set. What does it look like now, and how do you want it to look?

**Comment by meskarune** [comment]
This is standard x-term with the above added to .vimrc:

![terminalvimcolors](https://user-images.githubusercontent.com/153029/39448019-9d94c108-4c91-11e8-8067-6582d552783e.png)

the background in vim should be black - the default terminal background, but it's instead the default jellybeans background. I want the background color to always match whatever the terminal background color is.

**Comment by vinnyA3** [comment]
@meskarune Can you try adding it inside an `if` clause?

```
if has('termguicolors')
  set termguicolors
  let g:jellybeans_overrides = {
  \    'background': { 'ctermbg': 'none', '256ctermbg': 'none', 'guibg': 'none' },
  \}
  colorscheme jellybeans
endif
```

...This might do it for you :crossed_fingers:

**Comment by nanotech** [comment]
This should hopefully be addressed by the improved example in b02f1f34818d7eb559dfa8808607ca4c885c1eee. Go ahead and re-open or start a new issue if you're still having trouble.
